### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by jankeromnes**
Most typos were fixed using `codespell -w`.

### CODE REVIEW

File: doc/TUTORIAL.md [hunk 1]
```diff
@@ -218,11 +218,11 @@ Description of the code:
...
+4. We perform input validation by defining a schema which we expect the JSON we receive to conform to. This is done using [Joi](https://github.com/hapijs/joi). Defining a schema means we can ensure the JSON we receive meets our expectations and throw an error if we receive unexpected input without having to explicitly code validation checks. The schema also acts as a filter on the JSON object. Any properties we're going to reference need to be validated, otherwise they will be filtered out. In this case our schema declares that we expect to receive an object which must have a property called 'status', which is a string.
```
Comment (jankeromnes) [inline doc/TUTORIAL.md]: GitHub doesn't highlight the actual change in this line, so here it is in shorter form:

```diff
- to recieve an object
+ to receive an object
```

File: doc/TUTORIAL.md [hunk 2]
```diff
@@ -218,11 +218,11 @@ Description of the code:
...
+8. Our badge must implement the `async handle()` function. Because our URL pattern captures a variable called `gem`, our function signature is `async handle({ gem })`. We usually separate the process of generating a badge into 2 stages or concerns: fetch and render. The `fetch()` function is responsible for calling an API endpoint to get data. The `render()` function formats the data for display. In a case where there is a lot of calculation or intermediate steps, this pattern may be thought of as fetch, transform, render and it might be necessary to define some helper functions to assist with the 'transform' step.
```
Comment (jankeromnes) [inline doc/TUTORIAL.md]: GitHub doesn't highlight the actual change in this line, so here it is in shorter form:

```diff
- Our bage must
+ Our badge must
```

**Comment by paulmelnikow** [comment]
Thank you!

**Comment by jankeromnes** [comment]
Many thanks for the quick review @paulmelnikow ! 💯

**Comment by paulmelnikow** [comment]
If you're interested in some kinda fun yet kinda robotic work, we need to convert all of our `previewUrl`s to `staticPreview` + `pattern` + `namedParams`. There are just over 50 left so we're really in the home stretch. #2652 is one I did not long ago… happy to help if you're interested (we can chat on Discord or on #1961).

**Comment by jankeromnes** [comment]
@paulmelnikow Thanks for the suggestion! I opened #2772 with a bit more details, and my first roadblock (i.e. is there a way to expose the badge server on port 8080 for non-localhost access?)

**Pull request merged**